package.json

```diff
@@ -57,7 +57,7 @@
     "@types/angular": ">=1.5",
     "@types/lodash.frompairs": "^4.0.3",
     "@types/react": "^16.0.0",
-    "@types/react-dom": "^16.0.0",
+    "@types/react-dom": "16.0.1",
```

Comment:
Can this be `^16.0.1`?

Reply:
Nope. When I install `@types/react-dom@16.0.2` or later version it breaks the build, so we have to specify `16.0.1` 😇